src/app/components/transferDetailView/index.tsx

```diff
@@ -46,6 +54,49 @@ const ColumnContainer = styled.div({
   flexDirection: 'column',
 });
 
+const ScriptOutputHeadingText = styled.h1((props) => ({
+  ...props.theme.headline_s,
+  color: props.theme.colors.white[0],
+  marginBottom: props.theme.spacing(12),
+}));
+
+const ScriptText = styled.h1((props) => ({
+  ...props.theme.body_medium_m,
+  color: props.theme.colors.white[200],
+  width: 320,
+  wordWrap: 'break-word',
+}));
+
+const CrossContainer = styled.div({
+  display: 'flex',
+  marginTop: 10,
+  justifyContent: 'flex-end',
+  alignItems: 'flex-end',
+});
+
+const TransparentButton = styled.button({
+  background: 'transparent',
+  display: 'flex',
+  alignItems: 'center',
+  marginLeft: 10,
+});
+
+const ShowScriptBackgroundContainer = styled(animated.div)({
+  width: '100%',
+  height: '100%',
+  top: 0,
+  left: 0,
+  bottom: 0,
+  right: 0,
+  position: 'fixed',
+  zIndex: 10,
+  background: 'rgba(18, 21, 30, 0.8)',
```

Comment:
prefer to use theme.ts colors because when we do a rebrand, we don't want custom defined colors everywhere